ui/components/app/confirm-page-container/confirm-detail-row/index.scss

```diff
@@ -35,7 +35,7 @@
 
     text-transform: uppercase;
     margin-bottom: 6px;
-    color: var(--color-icon-default);
+    color: var(--color-text-default);
 
     &--edit {
       color: var(--color-primary-default);
```

Comment:
Updating semantically incorrect colors to correct text colors. See screenshot above